frontend/packages/knative-plugin/src/catalog/event-source-data.ts

```diff
@@ -38,7 +38,7 @@ export const getEventSourceCatalogProviderData = (
     },
     [EventSourceSinkBindingModel.kind]: {
       description: t(
-        'knative-plugin~Used to connect OpenShift managed applications like Deployments, StatefulSets, or Jobs to an event sink, for example, a Knative service, channel, or broker. SinkBinding is similar to a ContainerSource but works on existing OpenShift application resources, whereas the ContainerSource container lifecycle is fully managed by Openshift Serverless itself.',
+        'knative-plugin~Used to connect OpenShift managed applications like Deployments, StatefulSets, or Jobs to an event sink, for example, a Knative Service, Channel, or Broker. SinkBinding is similar to a ContainerSource but works on existing OpenShift Application resources, whereas the ContainerSource Container lifecycle is fully managed by OpenShift Serverless itself.',
```

Comment:
Channel and broker are not proper nouns so they don't need to be capitalized. Service is a proper noun however.

Reply:
Channel and Broker here as well are resources created via Serverless just Deployment is there in Kubernetes

Reply:
Thank you for clarifying this one as well, no changes needed then.